ui/components/MesheryApplications.js

```diff
@@ -481,16 +481,26 @@ function MesheryApplications({
         updateProgress({ showProgress : false })
         return;
       }
+      deleteApplication(id);
+    }
+
+    if (type === FILE_OPS.UPDATE) {
+      let response = await showUpdateModel("");
+      if (response=="No"){
+        updateProgress({ showProgress : false })
+        return;
+      }
       dataFetch(
-        `/api/application/${id}`,
+        `/api/application/${source_type}`,
         {
           credentials : "include",
-          method : "DELETE",
+          method : "PUT",
+          body : JSON.stringify({ application_data : { id, name, application_file : data }, save : true }),
         },
         () => {
-          console.log("ApplicationFile API", `/api/application/${id}`);
+          console.log("ApplicationFile API", `/api/application/${source_type}`);
           updateProgress({ showProgress : false });
-          enqueueSnackbar(`"${name}" Application deleted`, {
+          enqueueSnackbar(`"${name}" Application Updated Successfully`, {
```

Comment:
```suggestion
          enqueueSnackbar(`"${name}" Application Updated`, {
```

Reply:
Please don't use "successfully" in the snackbar messages, please also update the delete snackbar, it should show the app name when deleted. It was there previously.